src/enclave_proc/commands.rs

```diff
@@ -48,12 +48,24 @@ pub fn run_enclaves(
         .map_err(|e| e.add_subaction("Failed to construct CPU information".to_string()))?
         .get_cpu_config(args)
         .map_err(|e| e.add_subaction("Failed to get CPU configuration".to_string()))?;
+
+    let enclave_name = match &args.enclave_name {
+        Some(enclave_name) => enclave_name,
+        None => {
+            return Err(new_nitro_cli_failure!(
+                "Failed to set enclave name in the manager".to_string(),
+                NitroCliErrorEnum::FileOperationFailure
+            ))
+        }
+    };
```

Comment:
Perhaps I'm missing something, but does that mean that enclave_name is not an optional argument? I don't think it should be a required argument. Perhaps setting a useful default is better (eg, the enclave ID or https://github.com/fnichol/names).

Reply:
There is a default value to which the name is set in case the argument is missing, the name of the EIF (test.eif -> test). Still, this field is kept as an `Option` because at the moment of parsing the arguments it could be missing.